src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -155,6 +155,19 @@ public static ModuleCode parseModuleCode(String moduleCode) throws ParseExceptio
         }
         return new ModuleCode(trimmedModuleCode);
     }
+
+    /**
+     * Parses {@code Collection<String> moduleCodes} into a {@code Set<ModuleCode>}.
+     */
+    public static Set<ModuleCode> parseModuleCodes(Collection<String> moduleCodes) throws ParseException {
+        requireNonNull(moduleCodes);
+        final Set<ModuleCode> codeSet = new HashSet<>();
```

Comment:
May I ask what is the "final" modifier for? 😅

Reply:
The codeSet won't be changed (also followed the code from parseTags in the same file)

Reply:
Cool, thanks for the clarification!